src/CMakeLists.txt

```diff
@@ -2,63 +2,56 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ..)
 
 message(STATUS "CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
 
-find_package(EPICSV4)
-find_package(RdKafka REQUIRED)
-find_package(FlatBuffers REQUIRED)
-find_package(CLI11 REQUIRED)
-find_package(JSONForModernCPP REQUIRED)
-find_package(ConcurrentQueue REQUIRED)
-find_package(StreamingDataTypes COMPONENTS ce9a4745431e54bc45b0c927857a866fa77b84ec)
 find_package(CURL)
-find_package(fmt REQUIRED)
-find_package(GraylogLogger)
-find_package(StaticData COMPONENTS "schema-config-global.json")
+find_package(Threads)
 find_package(GitCommitExtract)
 
-set(path_include_common
-${CONCURRENTQUEUE_INCLUDE_DIR}
-${JSONFORMODERNCPP_INCLUDE_DIR}
-${CLI11_INCLUDE_DIR}
-${RDKAFKA_INCLUDE_DIR}
-${FLATBUFFERS_INCLUDE_DIR}
-${PROJECT_SOURCE_DIR}/src
-${PROJECT_BINARY_DIR}/src
-)
-
-set(path_include_common_suppressed_warnings
-        ${EPICSV4_INCLUDE_DIRS}
-        )
-
 set(libraries_common
-${path_library_epics_ca}
-${path_library_epics_pvData}
-${path_library_epics_pvAccess}
-${path_library_epics_NT}
-${RDKAFKA_LIBRARIES}
-${FETK_EXTRA_LIBRARIES}
-fmt::fmt
+        CONAN_PKG::librdkafka
+        CONAN_PKG::jsonformoderncpp
+        CONAN_PKG::concurrentqueue
+        CONAN_PKG::streaming-data-types
+        CONAN_PKG::cli11
+        CONAN_PKG::fmt
+        CONAN_PKG::flatbuffers
+        CONAN_PKG::asio
+        Threads::Threads
 )
 
-if (UNIX)
-  list(APPEND libraries_common pthread)
-endif()
+set(path_include_common "")
+
+# Allow a system installed EPICS library to be used instead of conan package (for ISIS-STFC)
+set(USE_EXTERNAL_EPICS FALSE CACHE BOOL "Look for system installed EPICS instead of using Conan package")
```

Comment:
I think we talked about this a while ago. Is there no possibility of getting STFC (Freddie I assume) to start using conan?

Reply:
Yes, we're using Conan for everything other than EPICS. The controls team want to use exactly the same version of EPICS for everything. Also several applications run on one machine on each instrument, and EPICS is bigger than all the other libraries combined so it does make the package to deploy a lot smaller without it.